### ISSUE TITLE
Feature request: Highlight the match under the cursor differently than other matches

### ISSUE THREAD

**Issue by scottchiefbaker** [RRethy/vim-illuminate#150]
I'm stuck on Vim 7.4 on a couple of servers so the [delay feature](https://github.com/RRethy/vim-illuminate#vim-users) isn't available to me. When I'm scrolling through a file with a lot of matches, vim-illuminate highlights ALL of them. Since delay is essentially 0ms it ends up highlighting/de-highlighting *lots of words* really quickly. This can make it easy to lose where the actual cursor is. 

It would be very helpful if the match under the cursor (primary match?) could be one color, and all the other matches on the screen a secondary color. This would allow me to keep an eye on where the cursor is when scrolling rapidly through a file.

**Comment by scottchiefbaker** [comment]
Yes this is exactly what I wanted. I dunno how I missed it in the documentation. I swear I read it before I opened the bug, but somehow I missed it.

Thanks for an EXCELLENT Vim plugin.

**Comment by Zwlin98** [comment]
It seems that illuminatedCurWord is not working in Neovim.
